### ISSUE TITLE
Training big-transformer model

### ISSUE THREAD

**Issue by tmtmaj** [pytorch/fairseq#3596]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?

I trained big-transformer model, but the result is worse than base model.
For big-transformer model, the BLEU score is 0.xx after 2 epoch.
I think that there are some problems about training setting such as learning rate, batch size, etc..
I also checked the loss through tensorboard. The result is
<img width="676" alt="image" src="https://user-images.githubusercontent.com/26248774/120985463-9a33e680-c7b6-11eb-99ef-4976939f1f7d.png">
Correct me if I'm doing something wrong.

#### Code

```
# pre-processing log

[src] Dictionary: 75100 types
[src] path/to/train.src: 22299663 sents, 423378845 tokens, 0.0% replaced by <unk>
[src] Dictionary: 75100 types
[src] path/to/valid.src: 965 sents, 22802 tokens, 0.0% replaced by <unk>
[src] Dictionary: 75100 types
[src] path/to/valid.src: 965 sents, 22802 tokens, 0.0% replaced by <unk>
[tgt] Dictionary: 75100 types
[tgt] path/to/train.tgt: 22299663 sents, 436780152 tokens, 0.0% replaced by <unk>
[tgt] Dictionary: 75100 types
[tgt] path/to/valid.tgt: 965 sents, 25795 tokens, 0.0% replaced by <unk>
[tgt] Dictionary: 75100 types
[tgt] path/to/valid.tgt: 965 sents, 25795 tokens, 0.0% replaced by <unk>
Wrote preprocessed data to path/to/data_dir
```

```
# use 8 V100 for training model

CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 fairseq-train path/to/data_dir \
    --arch transformer_wmt_en_de_big --share-all-embeddings --max-source-positions 256 --max-target-positions 256 \
    --optimizer adam --adam-betas '(0.9, 0.98)' --clip-norm 0.0 --activation-fn relu \
    --lr 0.0014 --lr-scheduler inverse_sqrt --warmup-updates 4000 --warmup-init-lr 1e-07 \
    --dropout 0.3 --attention-dropout 0.1 --weight-decay 0.0\
    --max-tokens 8000 --label-smoothing 0.1 --max-epoch 50 --log-interval 200\
    --keep-interval-updates -1 --save-interval-updates 0 --criterion label_smoothed_cross_entropy --update-freq 8\
    --save-dir path/to/save_dir --skip-invalid-size-inputs-valid-test
```

```
# fairseq-generate result

# 1th checkpoint
Generate valid with beam=5: BLEU4 = 1.05, 10.2/1.8/0.5/0.1 (BP=0.990, ratio=0.990, syslen=24590, reflen=24830)
# 2th checkpoint
Generate valid with beam=5: BLEU4 = 3.52, 22.0/6.0/2.5/1.1 (BP=0.818, ratio=0.833, syslen=20678, reflen=24830)
# 3th checkpoint
Generate valid with beam=5: BLEU4 = 8.87, 37.3/14.5/7.3/3.8 (BP=0.800, ratio=0.818, syslen=20300, reflen=24830)
# 4th checkpoint
Generate valid with beam=5: BLEU4 = 0.00, 0.8/0.0/0.0/0.0 (BP=0.000, ratio=0.078, syslen=1930, reflen=24830)
...
# ~50th checkpoint
BLEU4 = 0.xx
```
   

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master): fairseq repo at [40f6c75](https://github.com/pytorch/fairseq/commit/40f6c758b361adc7b87fa553f6f439daa4ee9501)
 - PyTorch Version (e.g., 1.0): torch==1.7.0+cu92
 - OS (e.g., Linux): Centos
 - How you installed fairseq (`pip`, source): pip 
 - Build command you used (if compiling from source): 
 - Python version: 3.6.5
 - CUDA/cuDNN version: 9.2
 - GPU models and configuration: 8 V100
 - Any other relevant information:

**Comment by fortunatekiss** [comment]
I'm facing the same issue as you, would you mind telling me how this problem ended up being resolved?
